src/ui/style/index.less

```diff
@@ -10,8 +10,15 @@
 
 html,
 body {
-  font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto,
-    'PingFang SC', 'Microsoft Yahei', sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto, sans-serif;
+  &[lang="zh_CN"], &[lang="zh_HK"] {
+    font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto,
+      'PingFang SC', 'Microsoft Yahei', sans-serif;
+  }
+  &[lang="jp"] {
+    font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto,
+      'Noto Sans JP', sans-serif;
+  }
```

Comment:
```suggestion
  font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto,
    'PingFang SC', 'Microsoft Yahei', sans-serif;
  &[lang='zh_CN'],
  &[lang='zh_HK'],
  &[lang='zh_CN'] body,
  &[lang='zh_HK'] body {
    font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto,
      'PingFang SC', 'Microsoft Yahei', sans-serif;
  }
  &[lang='jp'],
  &[lang='jp'] body {
    font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto,
      'Noto Sans JP', sans-serif;
  }```

Because the font defined by the body selector elsewhere will override this, as in the image below, the second character is not the correct Japanese character

<img width="410" alt="image" src="https://github.com/RabbyHub/Rabby/assets/20147371/9db1eade-07a1-4907-b0d5-4a9ad31c3a3a">

Reply:
<img width="410" alt="image" src="https://github.com/RabbyHub/Rabby/assets/20147371/9db1eade-07a1-4907-b0d5-4a9ad31c3a3a">

Reply:
I adopt your suggestion with small changes. It's weird because my change works for me...

Please check it again. Thanks!

Reply:
I came up with a smarter way to write it and pushed.